executor/executor.cc

```diff
@@ -593,18 +592,14 @@ void handle_completion(thread_t* th)
 		uint32_t* cover_count_pos = write_output(0); // filled in later
 
 		// Write out feedback signals.
-		// Currently it is code edges computed as xor of two subsequent basic block PCs.
 		uint64_t* cover_data = th->cover_data + 1;
 		uint32_t cover_size = th->kcov_size;
-		uint32_t prev = 0;
 		uint32_t nsig = 0;
 		for (uint32_t i = 0; i < cover_size; i++) {
 			uint32_t pc = cover_data[i];
-			uint32_t sig = pc ^ prev;
-			prev = hash(pc);
-			if (dedup(sig))
+			if (dedup(pc))
```

Comment:
If we do this, we need to drop everything related to signal from fuzzer, ipc and manager. If signal and cover are the same, we don't need to process/store them separately.
However, I am not sure it's a good idea to drop signal right now. Upstream kernel still does not work with clang, and we have older kernels that do not support clang. I think for now just need to use basic block coverage in clang to match gcc.